src/components/DocumentationTopic.vue

```diff
@@ -520,6 +539,13 @@ export default {
     declarations({ primaryContentSections = [] }) {
       return primaryContentSections.filter(({ kind }) => kind === SectionKind.declarations);
     },
+    hasOtherDeclarations({ declarations = [] }) {
+      // there's always only 1 `declaration` at this level
+      return declarations.length ? declarations[0].declarations.some(decl => Object.prototype.hasOwnProperty.call(decl, 'otherDeclarations')) : false;
+    },
+    declListToggleText({ declListExpanded }) {
+      return declListExpanded ? 'Hide other declarations' : 'Show all declarations';
```

Comment:
We should probably use the localized string JSON for this text, even if we don't yet have the non-English translations.

Reply:
I wanted to track it as an improvement in a separate bug if that's ok?

Reply:
For the translations specifically? I think it will make adding those later much easier since it will just be a configuration change at that point.

Reply:
Just move the strings to the lang files, ye. We can localise them later.